notNeededPackages.json

```diff
@@ -5635,6 +5635,10 @@
             "libraryName": "react-filepond",
             "asOfVersion": "7.1.0"
         },
+        "react-flags-select": {
+            "libraryName": "react-flags-select",
+            "asOfVersion": "2.0.2"
```

Comment:
I checked 2.0.2 on npm, and there don't seem to be any types. I think 2.0.3 might?

Reply:
Fixed.